Example/Pods/Pods.xcodeproj/project.pbxproj

```diff
@@ -1141,7 +1141,7 @@
 				GCC_PREFIX_HEADER = "Target Support Files/ShowTime/ShowTime-prefix.pch";
 				INFOPLIST_FILE = "Target Support Files/ShowTime/ShowTime-Info.plist";
 				INSTALL_PATH = "$(LOCAL_LIBRARY_DIR)/Frameworks";
-				IPHONEOS_DEPLOYMENT_TARGET = 8.2;
+				IPHONEOS_DEPLOYMENT_TARGET = 12.0;
```

Comment:
I think it's good that you support `12.0` here, but then why do the `Package.swift` supports `9`? IMHO the Package could support 12 as well, so you're good to go for the foreseeable years :) The real question is: If you say `12` here, the Package would crash on `9`, right?

Reply:
@KaneCheshire It's potentially an option to support iOS 9 in Xcode as well. Unsure if you'll get a warning. You decide :) The problem that I see for you only supporting iOS 9 is that I'm pretty sure Xcode 13 will not support anything older than iOS 12, by no means iOS 9.

Reply:
This is just the value set in the example project, not the lib itself. The `podspec` defines the minimum target so this is fine, it just means that the example project supports iOS 12